python/bids_import.py

```diff
@@ -196,14 +203,23 @@ def read_and_insert_bids(bids_dir, config_file, verbose, createcand, createvisit
             if(len(subproject_info) > 0):
                 subproject_id = subproject_info[0]['SubprojectID']
 
-        # greps BIDS session's info for the candidate from LORIS (creates the
-        # session if it does not exist yet in LORIS and the createvisit is set
-        # to true. If no visit in BIDS structure, then use default visit_label
-        # stored in the Config module)
+        visit_date = None
+        if startvisit:
+            scans_data = pd.concat([
+                scan_data.to_df()
+                for scan_data in bids_reader.bids_layout.get_collections('session', 'scans')
+            ])
+            visit_date = min(scans_data['acq_time'])
```

Comment:
In case scans_data['acq_time'] would be not defined, you could use the `get` function. 
ex: 
`scans_data.get('acq_time', datetime.datetime.now())`

Reply:
not sure I would use time of upload as `acq_time` here since a scan can be uploaded at a different date than the upload date. Might be best keeping the acquisition time as undef is that case instead of setting a default date. Unless I am misunderstanding your suggestion @xlecours?

Reply:
I makes perfect sense to have a date explicitly provided. I just don't know how it would work. Could it be grep'ed from the BIDS files?

Reply:
I believe this is what is being done in line 210. I guess the date could also be inside a session level file? I have not looked at the BIDS session spec in a while but it could be feasible. @laemtl thoughts?